doc/nrf/releases/release-notes-latest.rst

```diff
@@ -15,7 +15,7 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 Highlights
 **********
 
-There are no entries for this section yet.
+* Integrated Apple Find My supported for production.
```

Comment:
Please remove it. I just learned that highlights should not be contributed by us.

Reply:
yes, but this is actually one of the highlights approved by PMT so we can keep it here.